cli/run.ts

```diff
@@ -48,6 +48,12 @@ export async function createSite(config?: string): Promise<Site> {
   if (url) {
     log.info(`Loading config file <dim>${url}</dim>`);
     const mod = await import(url);
+    if (!mod.default) {
+      log.critical(
+        `[Lume] Missing Site instance! Ensure <cyan>_config.ts</cyan> does export the Site instance as default.`,
```

Comment:
Here you're assuming the configuration file is always `_config.ts`, which is not correct. It can be `_config.js` or any other filename or even an URL defined in the cli (`deno task build --config=my-lume-config.ts`).

We can say simply "Ensure your config file does export...". The path of the config file is already logged in the line 49.